jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1276,6 +1276,220 @@ ecma_builtin_typedarray_prototype_fill (ecma_value_t this_arg, /**< this argumen
   return ecma_copy_value (this_arg);
 } /* ecma_builtin_typedarray_prototype_fill */
 
+/**
+ * SortCompare abstract method
+ *
+ * See also:
+ *          ECMA-262 v5, 15.4.4.11
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+ecma_builtin_typedarray_prototype_sort_compare_helper (ecma_value_t j, /**< left value */
+                                                       ecma_value_t k, /**< right value */
+                                                       ecma_value_t comparefn) /**< compare function */
+{
+  ecma_value_t ret_value = ECMA_VALUE_EMPTY;
+  ecma_number_t result = ECMA_NUMBER_ZERO;
+  bool j_is_undef = ecma_is_value_undefined (j);
+  bool k_is_undef = ecma_is_value_undefined (k);
+
+  if (j_is_undef)
+  {
+    if (k_is_undef)
+    {
+      result = ECMA_NUMBER_ZERO;
```

Comment:
You can just use `return ecma_make_integer_value (0);`

If you use return, less else blocks and indentation is needed, which makes the code more readable.

Reply:
I've cleaned up the code a little bit since I don't need to check if `j` and `k` are undefined, see if that helps.